pkgs/by-name/ge/geo/package.nix

```
@@ -0,0 +1,32 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
+  pname = "geo";
+  version = "1.1";
+
+  src = fetchFromGitHub {
+    owner = "MetaCubeX";
+    repo = "geo";
+    rev = "v${version}";
+    hash = "sha256-lwFBevf3iP90LgnfUqweCjPBJPr2vMFtRqQXXUC+cRA=";
+  };
+
+  vendorHash = "sha256-FXvuojlMZRzi8TIQ2aPiDH7F9c+2dpe4PYzYWljfUIc=";
+
+  ldflags = [
+    "-s"
+    "-w"
+  ];
+
```

```suggestion

  passthru = {
    updateScript = nix-update-script { };
    tests.version = testers.testVersion { package = geo; };
  };

```

and a simple test and update script will help make this a bit more maintainable